intuita-webview/src/jobDiffView/DiffViewer/DiffItem.tsx

```diff
@@ -20,6 +21,7 @@ export const JobDiffView = ({
 	oldFileTitle,
 	newFileTitle,
 	title,
+	// @TODO why passing postMessage as prop?
 	postMessage,
```

Comment:
dependency injection is always better than direct global calls

Reply:
in react its very unusual to pass utility methods as  props instead of importing them

Reply:
and i think we need DI only when we want to be able to switch implementation, for instance, mock dependencies for unit testing. Does it make sense to inject utility functions that are static?